pom.xml

```diff
@@ -256,7 +256,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
       <artifactId>workflow-cps-global-lib</artifactId>
-      <version>2.10</version>
+      <version>2.14</version>
       <scope>test</scope>
```

Comment:
Darn, I have not noticed this is test dependency only so it will not enforce fixed version of `workflow-cps-global-lib` is used in production. :man_facepalming: